### DISCUSSION TITLE
No Process Id

### DISCUSSION THREAD

**Discussion by jessbr** [jamessimone/apex-rollup#258] [Q&A] [answered]
HI @jamessimone 
I tested Apex Rollups in Sandbox and it worked well. When I set it up in production and tried to run the onetime backfill rollups, when I clicked 'Start rollup!' the rollup job status was 'no process Id'. Do you know what this means? I have my flows and my CMDT set up the same way I did in Sandbox
<img width="946" alt="Apex Rollups" src="https://user-images.githubusercontent.com/63937108/156014969-8f70f993-e263-45ed-9b9d-e3042270bd2d.png">

Thanks!

**Comment by jamessimone** [comment, marked as answer]
@jessbr hi there! I would imagine that the `RollupSettings__c` custom setting still needs to have its org defaults set, based on that. Let me know if that does the trick.

**Comment by jessbr** [comment]
That is 100% correct. Thanks!

On Mon, Feb 28, 2022 at 2:32 PM James Simone ***@***.***>
wrote:

> @jessbr <https://github.com/jessbr> hi there! I would imagine that the
> RollupSettings__c custom setting still needs to have its org defaults
> set, based on that. Let me know if that does the trick.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jamessimone/apex-rollup/discussions/258#discussioncomment-2268201>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APHZUVGBGQZL44ISX77AYEDU5PEWHANCNFSM5PRPJM5Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Jessica Rosenberg, Consultant
North Peak <http://www.northpeak.com/> | @northpeaksol
<https://twitter.com/northpeaksol>
(858) 345-6173 ext. 107
Schedule a Meeting with me
<https://calendar.cirrusinsight.com/schedule/jessica-rosenberg>
Pronouns: She/Her
We're Hiring! <https://www.northpeak.com/careers/>

*Learn what it takes to become a CRM-centric nonprofit on our blog
<http://www.northpeak.com/blog/>.*
